main.go

```diff
@@ -71,6 +71,35 @@ func main() {
 		}
 	}
 
+	// The goroutine will continuously check bitcoind usage and auto close bitcoind if the client isn't active.
+	go func(checkInterval time.Duration) {
+		for {
+			autoCloseTime := time.Now().Add(time.Minute * time.Duration(-viper.GetInt("bitcoind_ctl.suspending_duration"))).Unix()
+			if controller.LastActiveTime.Unix() < autoCloseTime {
+				status, err := controller.getBitcoindStatus()
+				if err != nil {
+					log.WithError(err).Error("fail to auto check bitcoind status")
+					time.Sleep(checkInterval)
+					continue
+				}
+				statusNotready, _ := json.RawMessage(`{"error":"bitcoind process is not ready"}`).MarshalJSON()
```

Comment:
```suggestion
				statusNotReady := []byte(`{"error":"bitcoind process is not ready"}`)
```
https://play.golang.org/p/WJpiAm0rwSi